frontend/simple/views/SignUp.vue

```diff
@@ -70,14 +71,14 @@ export default {
     submit: function () {
       this.response = ''
       request.post(`${process.env.API_URL}/user/`)
-      .send(serialize(this.form, {hash: true}))
+      .send(serialize(this.form = this.$el.getElementsByTagName('form')[0], {hash: true}))
```

Comment:
`this.form` was set in `formHook`. Is `formHook` no longer being used / no longer needed? If so you can get rid of the related stuff below and above. I'm also hoping there's a cleaner way to grab the form object than `getElementsByTagName` (perhaps via a ref?).